bundles/redhat8.6/packages.txt.gotmpl

```diff
@@ -15,16 +15,18 @@ yum-utils
 cloud-init
 cloud-utils-growpart
 container-selinux
+{{ if .FetchKubernetesRPMs -}}
```

Comment:
we are keep adding flags for each new specific usecase: i.e. `--fetchKubernetesRPMs` , `--fetchKernelHeaders`, `--enableEus` etc. Thinking if there is any other way to reduce no of flags.

Reply:
fetchKubernetesRPMs not a flag. my bad.